Firestore/Swift/Source/Codable/DocumentReference+WriteEncodable.swift

```diff
@@ -26,68 +26,64 @@ extension DocumentReference {
   ///
   /// - Parameters:
   ///   - value: An instance of `Encodable` to be encoded to a document.
```

Comment:
These are great doc comments.

One quibble: the `value` here is of type `T`, which must conform to `Encodable`, but the type isn't actually `Encodable`. Should "`Encodable`" be replaced with "an `Encodable` type `T`"? I'm on the fence: it's more correct but definitely wordier.

What do you think?

Reply:
I'd leave it as-is...the exact type definition is available, a swift developer should be able to understand it?